odoo/tools/convert.py

```diff
@@ -31,6 +31,11 @@
 from .safe_eval import safe_eval as s_eval
 safe_eval = lambda expr, ctx={}: s_eval(expr, ctx, nocopy=True)
 
+
+# Match any QUOTED first cell of a row that starts with #, regardless of newlines
+INVALID_CSV_CELL_RE = re.compile(b'^\"\n*#', re.MULTILINE)
```

Comment:
Why the newlines? Does `csv` strip them when parsing the cell?

Also escaping the `"` is probably unnecessary, and the string should be marked as `raw` since it's a regex.

Reply:
> Does csv strip them when parsing the cell?

No, and it gets correctly treated as a cell value, this means that `test_invalid_10` and `test_invalid_20` are actually valid, my bad here